tutorial/lesson_12_using_the_gpu.cpp

```diff
@@ -309,7 +309,20 @@ Target find_gpu_target() {
         target.set_feature(Target::Metal);
     }
 #else
-    if (dlopen("libOpenCL.so", RTLD_LAZY) != NULL) {
+    if (void *handle = dlopen("libcuda.so", RTLD_LAZY)) {
+        int numDevices = 0;
+        int (*cuDeviceGetCount)(int *);
```

Comment:
Declare-and-initialize in one line is more appropriate.

Reply:
Fixed. Thank you!